cmd/trade.go

```diff
@@ -81,49 +82,49 @@ func init() {
 
 		if *fixedIterations == 0 {
 			fixedIterations = nil
-			log.Printf("will run unbounded iterations\n")
+			l.Info("will run unbounded iterations")
 		} else {
-			log.Printf("will run only %d update iterations\n", *fixedIterations)
+			l.Infof("will run only %d update iterations\n", *fixedIterations)
 		}
 	}
 
 	tradeCmd.Run = func(ccmd *cobra.Command, args []string) {
+		l := logger.MakeBasicLogger()
 		var botConfig trader.BotConfig
 		e := config.Read(*botConfigPath, &botConfig)
 		utils.CheckConfigError(botConfig, e, *botConfigPath)
 		e = botConfig.Init()
 		if e != nil {
-			log.Println()
-			log.Fatal(e)
+			logger.Fatal(l, e)
 		}
 
 		if *logPrefix != "" {
 			t := time.Now().Format("20060102T150405MST")
 			fileName := fmt.Sprintf("%s_%s_%s_%s_%s_%s.log", *logPrefix, botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB, t)
 			e = setLogFile(fileName)
 			if e != nil {
-				log.Println()
-				log.Fatal(e)
+				l.Info("")
```

Comment:
can remove the empty line printout here -- logger.Fatal starts off by printing an empty line